### PR TITLE
724 ac community structure

### PR DESCRIPTION
**Pull request by annacanal**
- [x] CommunityStructure
- [x] Modularity

Also some updates in documentation in several measures

### CODE REVIEW

File: braph2/graph/measures/Modularity.m [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            %
+            % See also getClass, getName.
+        
+            description = [ ...
+                'The modularity of a graph is the quality' ...
+                'of the resulting partition of the network. ' ...
+                ];
+        end
+        function available_settings = getAvailableSettings()
+            % GETAVAILABLESETTINGS returns the setting available to Modularity
```
Comment (maverickyuwei) [inline braph2/graph/measures/Modularity.m:84]: should it be lowercase 'm'odularity?

Comment (annacanal) [reply braph2/graph/measures/Modularity.m:84]: According to Degree measure, we wrote here the name of the measure class, so in this case: Modularity

Comment (maverickyuwei) [reply braph2/graph/measures/Modularity.m:84]: I see, the same pattern as community structure as well!

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
errors in unittest... see skype

**Pull request merged**